### DISCUSSION TITLE
Orbit File Was Not Found

### DISCUSSION THREAD

**Discussion by Rofiatulumami** [alexisInSAR/EZ-InSAR#80] [Help]
Hi, excuse me i wanna ask about this error during running the "Pre-run of ISCE Processing" they said, orbit was not found in my folder and then the run_files directory is not detected. What should i do to fix this? 
![Screenshot from 2024-02-29 14-11-37](https://github.com/alexisInSAR/EZ-InSAR/assets/161690953/ec6ac5bf-fbed-49b0-a5cf-7352a43631b0)

**Comment by alexisInSAR** [comment]
Hi @Rofiatulumami, 

I am preparing the next version of EZ-InSAR. The issue will the ASF server will be solved. 

Kind regards, 

Alexis
